### PR TITLE
Milestone: Storing data into local storage with javascript

### PR DESCRIPTION
**Pull request by yaredtekle022**
**This list of activities was done**

> Edited index.html file

> Added localstorage.js file containing the functionality for this project

> set linters

### CODE REVIEW

**Review by levy002 — COMMENTED** [review]
## Hello @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle:  :red_circle: 

You have submitted a project, that doesn't run linters. Kindly make sure your linter is running and all passed and submit again for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: localstorage.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    return;
+  }
+
+  // saving inputs
+  const userInfo = {
+    username,
+    email,
+    textarea,
+  };
+
```
Comment (BertrandConxy) [inline localstorage.js]: - [ ] Good job so far 👏  As an improvement to what you have done, It is important to note that the data are saved in the local storage every time the input value changes. That means the event that functions have to depend on is `OnChange` not `OnSubmit` as you have done. But you are almost there. I can see that data save when I click Get in touch button and retrieved back when I reload the page. So please work on the suggested improvement to complete it 😃 
image

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @yaredtekle022, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @yaredtekle022 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Data saved in local storage :heavy_check_mark: 
Made  requested changes :heavy_check_mark: 
All linters are passing :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**